__tests__/app.test.js

```diff
@@ -37,65 +37,103 @@ describe("/api/categories", () => {
+        });
+      });
+  });
+  test("status 400, if given a invalid review id", () => {
+    return request(app)
+      .get("/api/reviews/banana")
       .expect(400)
-      .then(({body}) =>{
+      .then(({ body }) => {
         expect(body.msg).toBe("bad request");
-      })
-    })
-    test("status 404, if given a valid review id that doesn't exist", () =>{
-      return request(app)
-      .get ("/api/reviews/1000")
+      });
+  });
+  test("status 404, if given a valid review id that doesn't exist", () => {
+    return request(app)
+      .get("/api/reviews/1000")
       .expect(404)
-      .then(({body}) =>{
+      .then(({ body }) => {
         expect(body.msg).toBe("review does not exist");
-      })
-    })
-})
+      });
+  });
+});
+
+describe("/api/reviews", () => {
+  test("status 200, returns all reviews", () => {
+    return request(app)
+      .get("/api/reviews")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.reviews.length).toBe(13);
+        body.reviews.forEach((reviews) => {
+          expect(reviews).toMatchObject({
+            review_id: expect.any(Number),
+            title: expect.any(String),
+            designer: expect.any(String),
+            owner: expect.any(String),
+            review_img_url: expect.any(String),
+            review_body: expect.any(String),
+            category: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+            comment_count: expect.any(Number),
+          });
+        });
+      });
+  });
+});
 
-describe("/api/reviews", () =>{
-    test("status 200, returns all reviews", () =>{
-        return request(app)
-        .get("/api/reviews")
-        .expect(200)
-        .then(({body}) =>{
-            expect(body.reviews.length).toBe(13);
-            body.reviews.forEach((reviews) => {
-                expect(reviews).toMatchObject({
-                    review_id: expect.any(Number),
-                    title: expect.any(String),
-                    designer: expect.any(String),
-                    owner: expect.any(String),
-                    review_img_url: expect.any(String),
-                    review_body: expect.any(String),
-                    category: expect.any(String),
-                    created_at:expect.any(String),
-                    votes: expect.any(Number),
-                    comment_count: expect.any(Number)
-                });
-              });
-        })
-    })
-})
+describe("/api/reviews/:review_id/comments", () => {
```

Comment:
This is fine but I think you might be missing a couple of test cases,
What if the review has no comments, this should be an empty array right? 
You're currently not testing for this :)